openlibrary/templates/merge/authors.html

```diff
@@ -48,33 +48,33 @@ <h1>$_("Merge Authors")</h1>
         <div class="note">$_("Please select some authors to merge.")</div>
 
 <ol class="sansserif">
-    <li>Select a &quot;master&quot; record that best represents the author -
+    <li>$_('Select a &quot;master&quot; record that best represents the author -')
```

Comment:
I don't think those will get rendered correctly? I think using `$` instead of `$:` will cause this to get rendered literally? But not sure.

```suggestion
    <li>$_('Select a "master" record that best represents the author -')
```

Reply:
Confirmed this is the case.